includes/class-sensei-frontend.php

```diff
@@ -1705,6 +1707,10 @@ public function sensei_process_registration() {
 		$new_user_email    = $_POST['sensei_reg_email'];
 		$new_user_password = $_POST['sensei_reg_password'];
 
+		if ( isset( $_POST['sensei_reg_http_referer'] ) && '' !== $_POST['sensei_reg_http_referer'] ) {
+			$new_user_http_referer = esc_url_raw( wp_unslash( $_POST['sensei_reg_http_referer'] ) );
```

Comment:
Is `wp_unslash` necessary?

The URL is also being double escaped. It's being escaped here and on line 1775. It's always better to escape late, so I would do the escaping further down.

Reply:
I had to add `wp_unslash` and the escaping here to pass the Travis CI build. Those were the issues that blocked my first attempt... 
I found [this article](https://github.com/WordPress/WordPress-Coding-Standards/wiki/Fixing-errors-for-input-data) that explains why we should use the `wp_unslash` for $_POST, so I think that's why the sniffer marked that as a violation.
So, what is your suggestion? Leave it like this? Add the sniffer ignore comment? Or something else?